src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -193,16 +200,31 @@ const Contact: React.FC<Props> = ({
                 </NativeSelect>
               </Box>
             </Grid>
-            <Grid item xs={12} lg={4}>
+            <Grid item xs={12} md={6} lg={4}>
               <Box className={classes.boxBottom}>
-                <TextField
-                  variant="outlined"
-                  size="small"
-                  style={{ width: '100%' }}
-                  placeholder="Currency" //TODO: change to select with currency options
-                  value={values.amountCurrency}
-                  onChange={(event) => handleChange(event, 'amountCurrency')}
-                />
+                <InputLabel id="currency-label">{t('Currency')}</InputLabel>
+                <Select
+                  labelId="currency-label"
+                  label={t('Currency')}
+                  placeholder="Currency"
+                  data-testid="pledgeCurrency"
+                  inputProps={{
+                    'data-testid': 'pledgeCurrency-input',
+                  }}
+                >
+                  <MenuItem value={'Currency'} disabled>
+                    {t('Currency')}
+                  </MenuItem>
+                  {constants?.constant?.pledgeCurrency &&
+                    getPledgeCurrencyOptions(
+                      constants?.constant?.pledgeCurrency,
+                      'short',
+                    )}
```

Comment:
If you did the change I mentioned in the code above, you can change this to the below.
```suggestion
{getPledgeCurrencyOptions(pledgeCurrencies, 'short')}
```

Reply:
This makes the `src/lib/getCurrencyOptions.tsx` do the optional chaining for us